app/paths.js

```diff
@@ -180,7 +180,9 @@ module.exports = {
         index: '/settings/email-notifications'
       },
       teamMembers: {
-        index: '/settings/team-members'
+        index: '/settings/team-members',
+        delete: '/settings/team-members/:externalUserId/delete',
+        permission: '/settings/team-members/:externalUserId/permission'
```

Comment:
`externalUserId` is this the actual key of the returned user id from adminusers?

Reply:
not sure I follow you .... it's the external user id as shown in the adminusers table, as far as I understand it